src/contracts/acrossConfigStore.e2e.ts

```diff
@@ -17,7 +17,7 @@ describe("AcrossConfigStore", function () {
   });
   test("getL1TokenConfig", async function () {
     const result = await client.getL1TokenConfig(wethAddress);
-    assert.ok(result.transferThreshold);
+    assert.ok(result.rateModel);
```

Comment:
Do we want to test for some of the additional fields that we expect to be included in `l1TokenConfig` ?

Reply:
This is the only required param in `l1TokenConfig` at the moment